cdc/model/mounter.go

```diff
@@ -13,14 +13,16 @@
 
 package model
 
-import "context"
+import (
+	"context"
+)
 
 // PolymorphicEvent describes a event can be in multiple states
 type PolymorphicEvent struct {
-	Ts       uint64
-	RawKV    *RawKVEntry
-	Row      *RowChangedEvent
-	finished chan struct{}
+	Ts       uint64           `json:"t"`
+	RawKV    *RawKVEntry      `json:"-"`
+	Row      *RowChangedEvent `json:"r"`
+	Finished chan struct{}    `json:"-"`
```

Comment:
why make `Finished` public? and why not store `RowChangedEvent` to the disk directly

Reply:
by the way, i think `json` is not a suited serialization algorithm, it's advantages is readability but we don't need it here

Reply:
Good catch, `Finished` is set to public by mistake. And use gob encode to store PolymorphicEvent to file